registration/tests/admin_approval_backend.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from registration.models import SupervisedRegistrationProfile
+from registration.backends.admin_approval.views import RegistrationView
+from registration.users import UserModel
+from .default_backend import DefaultBackendViewTests
+
+
+class AdminApprovalBackendViewTests(DefaultBackendViewTests):
+    """
+    Test the admin_approval registration backend.
+
+    Running these tests successfully will require two templates to be
+    created for the sending of activation emails; details on these
+    templates and their contexts may be found in the documentation for
+    the default backend.
+
+    """
+    urls = 'test_app.urls_admin_approval'
```

Comment:
Replace this with `@override_settings(ROOT_URLCONF=...)` as done in 80c1604c672a4cf382185103e5df9d2958563852.